src/elements/content-uploader/__tests__/ProgressBar.test.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import ProgressBar from '../ProgressBar';
```

Comment:
which component does this test?

Reply:
The progressbar when a item is being uplaoded via the contentuploader